### ISSUE TITLE
Common AppData dir warnings (Trac #47)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#51]
The "devices/plugins does not exist in common AppData dir" warnings pop up on program launch if the user does not have Microdrop installed on their system. These warnings should only be displayed if the application is being run from an exe file (otherwise, the devices and plugins directories should exist in the src tree).

Migrated from http://microfluidics.utoronto.ca/ticket/47
```json
{
    "status": "closed", 
    "changetime": "2015-01-07T21:59:40", 
    "description": "The \"devices/plugins does not exist in common AppData dir\" warnings pop up on program launch if the user does not have Microdrop installed on their system. These warnings should only be displayed if the application is being run from an exe file (otherwise, the devices and plugins directories should exist in the src tree).", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1420667980075591", 
    "component": "core", 
    "summary": "Common AppData dir warnings", 
    "priority": "minor", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-02-20T03:35:38", 
    "milestone": "Microdrop 1.0", 
    "owner": "ryan", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120801T15:50:46: ryan changed milestone from "0.2" to "0.3"

**Comment by cfobel** [comment]
Trac update at 20140313T20:00:48: ryan changed milestone from "1.1" to ""

**Comment by cfobel** [comment]
Trac update at 20150105T18:07:28: 
* cfobel changed milestone from "" to "Microdrop 1.0"
* cfobel changed status from "new" to "assigned"

**Comment by cfobel** [comment]
Trac update at 20150107T21:59:03: ryan commented:

> Fixed in commit [changeset:f79a733/microdrop].

**Comment by cfobel** [comment]
Trac update at 20150107T21:59:40: 
* ryan changed resolution from "" to "fixed"
* ryan changed status from "assigned" to "closed"
